src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -34,6 +34,26 @@ internal class UserDefinedFunction : TexlFunction
 
         public TexlNode UdfBody { get; }
 
+        public int GetArgIndex(string argName)
+        {
+            var argIndex = -1;
+
+            if (string.IsNullOrEmpty(argName))
+            {
+                return argIndex;
+            }
+
+            foreach (var arg in _args)
+            {
+                if (arg.VarIdent.Name.Value == argName)
+                {
+                    argIndex = arg.ArgIndex;
```

Comment:
>argIndex = arg.ArgIndex [](http://example.com/codeflow?start=20&length=23)

return?

Could there be multiple args with the same name?

Reply:
There can be, but rule will be erroneous, in that case we won't even get here.

Reply:
great then we can return early when the condition is true

Reply:
Done!